src/universalinit/templates/flask/config.yml

```
@@ -14,7 +14,7 @@ init_files: []
 init_minimal: "Minimal Flask application initialized"
 
 run_tool:
-  command: "./start.sh"
+  command: "source venv/bin/activate && flask run --host 0.0.0.0 --port=<port> --reload"
```

Here should be a space instead of the `=` for standardization?
```suggestion
  command: "source venv/bin/activate && flask run --host 0.0.0.0 --port <port> --reload"
```